### PR TITLE
Make test output deterministically ordered and easier to read

### PR DESCRIPTION
**Pull request by tamird**
@mrtracy

### CODE REVIEW

File: storage/client_event_test.go [hunk 1]
```diff
@@ -33,12 +35,14 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+const numberOfProtoMethods = 30
```
Comment (bdarnell) [inline storage/client_event_test.go]: Where did this come from? If we need a constant for this it should come from the proto package where the method enum is defined.

File: storage/client_event_test.go [hunk 2]
```diff
@@ -33,12 +35,14 @@ import (
...
+
 type storeEventReader struct {
 	// If true, Update events will be recorded in full detail. If false, only a
 	// count of update events will be recorded.
 	recordUpdateDetail  bool
-	perStoreFeeds       map[proto.StoreID][]string
-	perStoreUpdateCount map[proto.StoreID]map[proto.Method]int
+	perStoreFeeds       [][]string
+	perStoreUpdateCount [][numberOfProtoMethods]int
```
Comment (bdarnell) [inline storage/client_event_test.go]: This feels wrong; I'd rather keep this a map and sort the results before printing them (but grumble generics grumble grumble)

Comment (tamird) [inline storage/client_event_test.go]: I feel like sorting the keys is seriously a lot of shenanigans, but i'll try it tomorrow

**Comment by tamird** [comment]
@bdarnell PTAL addressed your concern

**Comment by mrtracy** [comment]
LGTM, didn't even know about tabwriter

**Comment by bdarnell** [comment]
LGTM

**Pull request merged**